File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SpellwovenJacket.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.MediumArmor;

/* JADX INFO: loaded from: classes3.dex */
public class SpellwovenJacket extends MediumArmor {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.armor_medium_spellwoven_jacket_name;
        this.idDescription = R.string.armor_medium_spellwoven_jacket_description;
        this.idImage = R.drawable.spellwoven_jacket;
        this.price = 741L;
        this.maxHp = 200;
        this.constitution = 10;
        this.dexterity = 10;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SpellwovenJacket.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.MediumArmor

class SpellwovenJacket : MediumArmor() {
    override fun configureProperties() {
        idName = R.string.armor_medium_spellwoven_jacket_name
        idDescription = R.string.armor_medium_spellwoven_jacket_description
        idImage = R.drawable.spellwoven_jacket
        price = 741L
        maxHp = 200
        constitution = 10
        dexterity = 10
    }
}
